tests/test_point.py

```diff
@@ -113,19 +113,6 @@ def test_point(self):
         self.assertEqual(p.__geo_interface__,
                          {'type': 'Point', 'coordinates': (3.0, 4.0)})
 
-    @shapely20_deprecated
-    def test_point_mutate(self):
```

Comment:
Similar recommendation as above: rename this `test_point_immutable` and check that `.coord` raise TypeError.